metadata-ingestion/tests/unit/test_transform_dataset.py

```diff
@@ -815,7 +815,7 @@ def test_pattern_dataset_tags_transformation(mock_time):
         )
     )
 
-    assert len(outputs) == 3
+    assert len(outputs) == 5
```

Comment:
we should also be checking the actual contents of the tag creations here

Reply:
done